src/System.Memory/src/System.Memory.csproj

```diff
@@ -0,0 +1,77 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <StrongNameKeyId>Open</StrongNameKeyId>
+    <Nullable>disable</Nullable>
+    <IsPackable>false</IsPackable>
+    <VersionPrefix>4.5.5</VersionPrefix>
+    <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.6.0</VersionPrefix>
+    <AssemblyVersion>4.0.1.2</AssemblyVersion>
+    <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.1.2</AssemblyVersion>
+    <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.0.2.0</AssemblyVersion>
+    <PackageValidationBaselineVersion>4.5.5</PackageValidationBaselineVersion>
+    <IsPartialFacadeAssembly Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</IsPartialFacadeAssembly>
+    <DefineConstants Condition="'$(IsPartialFacadeAssembly)' != 'true'">$(DefineConstants);netstandard;FEATURE_PORTABLE_SPAN</DefineConstants>
+    <ILLinkClearInitLocals>true</ILLinkClearInitLocals>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) != '.NETFramework'">
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(CommonPath)\CoreLib\System\Buffers\Text\FormattingHelpers.CountDigits.cs" Link="Common\CoreLib\System\Buffers\Text\FormattingHelpers.CountDigits.cs" />
+    <Compile Include="$(CommonPath)\System\MutableDecimal.cs" Link="Common\System\MutableDecimal.cs" />
+    <Compile Include="$(CommonPath)\System\Numerics\Hashing\HashHelpers.cs" Link="Common\System\Numerics\Hashing\HashHelpers.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
```

Comment:
I wonder if we gain anything from keeping these under `$(CommonPath)\CoreLib` they were under that path in corefx because the code was actually shared with the System.Private.CoreLib build.  That won't be the case here, so we could just move these under `System.Memory` and simplify this.

Reply:
I see that MutableDecimal.cs was also only used by System.Memory. Although it wasn't under CoreLib, I can also move it to keep Common a bit cleaner.

Reply:
Reactivating this since I still see these in `Common` - For any source files that make up only the System.Memory functionality we should not have them in `Common`.  You should be able to move everything this PR adds to Common under `System.Memory\src`.  Of all of those the only one that *might* need to be shared in the future is `HashHelpers`, I see `MutableDecimal` is also used by the tests - but that would be OK to pull from System.Memory\src.

Reply:
Ah, I misread the first comment. I'll move the files from Common to System.Memory/src.